.eslintrc.json

```diff
@@ -125,18 +126,19 @@
       "files": ["*.ts"],
       "parser": "@typescript-eslint/parser",
       "plugins": [
-        "@typescript-eslint",
-        "eslint-plugin-tsdoc"
+        "@typescript-eslint"
       ],
       "extends": ["airbnb-typescript/lib/shared"],
       "rules": {
         "@typescript-eslint/no-implied-eval": "off",
         "@typescript-eslint/no-throw-literal": "off",
         "comma-dangle": ["warn", "never"],
+        "import/first": "off",
```

Comment:
This seems useful, why disable it?

Reply:
I believe it was because it was complaining about the order of `import x from 'x';` vs `import y = require('y');`, which shouldn't matter. I re-enabled the rule and didn't get any errors this time, so I must have refactored out the code that it was complaining about. I'll leave it re-enabled for the time being.